examples/sandbox/json/ethereum/ethSignTypedDataV4.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
I grabbed this payload straight from the OpenSea request to its own backend server, which was responding with "invalid order signature"